content/app/github/window-helper.js

```diff
@@ -33,6 +33,15 @@ define("content/app/github/window-helper", function(require, exports, module) {
 				return new Box(0, 0, width, height);
 			},
 
+			/**
+			 * Scrolls vertically document body.
+			 * @param  {Number}		scrollDelta
+			 */
+			scrollTop : function(scrollDelta) {
```

Comment:
Beside that what happen is caller want to scroll go position that is smaller than minimum scrollTop or greater than maximum scrollTop?